template/python3-flask-debian/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM openfaas/of-watchdog:0.7.2 as watchdog
+FROM python:3-slim-buster
+
+COPY --from=watchdog /fwatchdog /usr/bin/fwatchdog
+RUN chmod +x /usr/bin/fwatchdog
+
+ARG ADDITIONAL_PACKAGE
+# Alternatively use ADD https:// (which will not be cached by Docker builder)
+
+RUN apt-get update && apt-get --assume-yes install musl-dev gcc make ${ADDITIONAL_PACKAGE}
```

Comment:
Why are we adding `musl-dev`? The whole point of using Debian is to avoid musl and to benefit from all the packages built with glibc.

Reply:
More used to seeing `-qy`, how does `--assume-yes` differ?